manifests/r/Rosalie/Rosalie'sMupenGUI/0.5.7/Rosalie.Rosalie'sMupenGUI.yaml

```
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: Rosalie.Rosalie'sMupenGUI

```

```suggestion
PackageIdentifier: Rosalie.RosaliesMupenGUI
```

- If the suggestion above is taken, please update file name and path to match this change.